org-tree-slide.el

```diff
@@ -238,6 +238,8 @@ nil: keep the same position."
   "A hook run before moving to the previous slide.")
 (defvar org-tree-slide-before-content-view-hook nil
   "A hook run before showing the content.")
+(defvar org-tree-slide-content-pos nil
```

Comment:
This variable should be defined as an internal, could you please replace this with `org-tree-slide--content-pos`?